testcases/RunHostTestpVM.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python2
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2018
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+'''
+RunHostTestpVM
+--------------
```

Comment:
can we handle this case aswell in `RunHostTest` itself and based on bmc_type we can do a switch to VM and NV code path...that would ease to run same testcases in both platforms in future...

Reply:
Sure, doing the same.